ospd_openvas/daemon.py

```diff
@@ -626,7 +626,7 @@ def get_dependencies_vt_as_xml_str(
             _vt_dep = Element('dependency')
             try:
                 _vt_dep.set('vt_id', dep)
-            except TypeError:
```

Comment:
This doesn't seem to be about improving the tests.

Reply:
You are right. This improve the error handling, not the test. But it is related to the first commit. I should have done this with the first commit or in a separated one.